examples/js/archive.js

```diff
@@ -383,6 +359,60 @@ shareMapBtn.addEventListener('click', () => {
   bootstrap.Modal.getInstance(shareModal).show()
 });
 
+// aggregate coordinates of all images into an array
+function getCornerBounds(imgCollection) {
+  let cornerBounds = []; 
+
+  // aggregate coordinates for multiple images int cornerBounds
+  if (imgCollection.length > 1) { 
+    imgCollection.forEach((imageObj) => {
+      if (imageObj.nodes) {
+        for(let i = 0; i < imageObj.nodes.length; i++) {
+          let corner = [];
+          corner[0] = imageObj.nodes[i].lat;
+          corner[1] = imageObj.nodes[i].lon;
+          cornerBounds.push(corner); // then we have array of arrays e.g., [ [lat, lon], [..], [..], [], [], [], [], [] ] for two images etc...
+        }        
+      }
+    });
+  } else { // aggregate coordinates for a single image into cornerBounds
+    if (imgCollection[0].nodes) {
+      for(let i = 0; i < imgCollection[0].nodes.length; i++) {
+        let corner = [];
+        corner[0] = imgCollection[0].nodes[i].lat;
+        corner[1] = imgCollection[0].nodes[i].lon;
+        cornerBounds.push(corner); // then we have [ [lat, long], [..], [..], [..] ] for just one image...
+      } 
+    }
+  }
+
+  return cornerBounds;
+}
+
+// converts legacy json objects (from mapknitter.org) to working format
+function transform(imgObj) {
```

Comment:
This looks good; what about a name like `updateLegacyJson(json)` as a name?

Reply:
Okay, that's fine.